### PR TITLE
WIP update README

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,20 @@
-# .github
+# Welcome! 
 
-Contains community health files as described in https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file
+Hello, and thank you for visiting the AgPipeline project. This page gives you all the basic information you need to know about why this project exists, who it is for, and how to give back with your expertise. More technical documentation is available on [the project website](https://agpipeline.github.io/). 
 
+# Project purpose
 
+The need that this project fulfills is to provide a scalable, modular, extensible pipeline to process sensing and image data. Most of the existing software for this purpose is expensive and proprietary, or built for very specific projects. Our pipeline is intended for a diverse array of users, from agricultural researchers to engineers, to be able to use, and to implement their own algorithms within this framework.
```
Comment (Chris-Schnaufer) [inline README.md]: Perhaps change second sentence to: Most of the existing software is built for very specific projects and is therefore expensive and proprietary.

**Review by Chris-Schnaufer — APPROVED** [review]
Put each sentence on a separate line?

File: README.md (hunk 1 above)
Comment (KristinaRiemer) [reply README.md]: I thought that there is proprietary software for doing image processing that is more general purpose and can be used for lots of different projects?

Comment (Chris-Schnaufer) [reply README.md]: That is true for some tools and their pipelines, such as Lidar transformations and its manipulation, and RGB data. Once you get beyond those specific manipulations there's a tendency to become custom. Commercial packages that address the larger workflows tend towards the expensive side. Does this help?

Comment (KristinaRiemer) [reply README.md]: Yup! Thanks for the explanation.

**Comment by KristinaRiemer** [comment]
I've implemented Chris' suggestions.

**Pull request merged**